### PR TITLE
DM-31801: Use lsst prefix for logger

### PR DESCRIPTION
**Pull request by timj**

### CODE REVIEW

File: python/lsst/meas/deblender/baseline.py [hunk 1]
```diff
@@ -725,7 +725,7 @@ def newDeblend(debPlugins, footprint, mMaskedImage, psfs, psfFwhms,
     if log is None:
         import lsst.log as lsstLog
 
-        component = 'meas_deblender.baseline'
+        component = 'lsst.meas_deblender.baseline'
```
Comment (leeskelvin) [inline python/lsst/meas/deblender/baseline.py:728]: Not sure if you want to switch to using double-quotes here (and elsewhere in this PR), for consistency with other strings? Not a major concern of mine however.

**Review by leeskelvin — APPROVED** [review]

File: python/lsst/meas/deblender/baseline.py (hunk 1 above)
Comment (timj) [reply python/lsst/meas/deblender/baseline.py]: Actually. It's not the right logger name. It should be `meas.deblender` and not `meas_deblender` so I fixed it to use `__name__` (Which is what it should have been all along.

Comment (timj) [reply python/lsst/meas/deblender/baseline.py]: I'd rather not go through and unify strings everywhere on this PR at this time.

Comment (leeskelvin) [reply python/lsst/meas/deblender/baseline.py]: Ah, a good move switching to `__name__`, thanks!

**Pull request merged**